Now shows the Tweets as embedded Tweets on clicking markers

prasht63 opened the pull request:
![image](https://cloud.githubusercontent.com/assets/2105062/8257955/f266a26a-16ce-11e5-929c-123cb59abe9f.png)

sudheesh001 commented:
@prasht63 The twitter brand policy says don't duplicate the bird as well as to keep a 150% buffer space. I am not sure if we should be using the twitter icon as our markers.
https://about.twitter.com/press/twitter-brand-policy
@mariobehling Please advise if this is acceptable or if I am headed into the wrong direction.

prasht63 commented:
@sudheesh I had removed the Twitter Bird marker its just in the screen
grab.I have replaced them with a normal marker

On Sat, Jun 20, 2015 at 12:28 AM, Sudheesh Singanamalla <
notifications@github.com> wrote:

> @prasht63 <https://github.com/prasht63> The twitter brand policy says
> don't duplicate the bird as well as to keep a 150% buffer space. I am not
> sure if we should be using the twitter icon as our markers.
> https://about.twitter.com/press/twitter-brand-policy
> @mariobehling <https://github.com/mariobehling> Please advise if this is
> acceptable or if I am headed into the wrong direction.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loklak/loklak_webclient/pull/202#issuecomment-113606534>
> .
>

sudheesh001 commented:
Looks good and works. Good job @prasht63
